src/Scaffold/Modes/AppendMode.php

```
@@ -0,0 +1,53 @@
+ * Applies a scaffold file by appending its content to an existing destination, or writing it fresh.
+ *
+ * @copyright Copyright (C) 2025 Terabytesoftw.
+ * @license https://opensource.org/license/bsd-3-clause BSD 3-Clause License.
+ */
+final class AppendMode implements ModeInterface
+{
+    public function apply(
+        FileMapping $mapping,
+        string $projectRoot,
+        Hasher $hasher,
+        string|null $hashAtScaffold,
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$hashAtScaffold'. (undefined)

(UnusedFormalParameter)